main.tf

```diff
@@ -16,10 +16,12 @@
 
 locals {
   stdlib_head = "${file("${path.module}/files/startup-script-stdlib-head.sh")}"
+  gsutil_el   = "${var.enable_init_gsutil_crcmod_el == "false" ? "" : file("${path.module}/files/init_gsutil_crcmod_el.sh")}"
   stdlib_body = "${file("${path.module}/files/startup-script-stdlib-body.sh")}"
   # List representing complete content, to be concatenated together.
   stdlib_list = [
     "${local.stdlib_head}",
+    "${local.gsutil_el}",
```

Comment:
This is how the fragments are assembled from the feature flags exposed to the user as input vars.